docs/reverse-proxy.md

```diff
@@ -34,8 +34,8 @@ Apache virtual server configuration
         ProxyRequests           Off
         ProxyPreserveHost       On
         AllowEncodedSlashes     NoDecode
-        ProxyPass               /       http://127.0.0.1:4873 nocanon
-        ProxyPassReverse        /       http://127.0.0.1:4873
+        ProxyPass               /       http://127.0.0.1:4873/ nocanon
+        ProxyPassReverse        /       http://127.0.0.1:4873/
     </VirtualHost>
     </IfModule>
 ````
```

Comment:
Could you add a link to https://github.com/verdaccio/docker-examples/tree/master/apache-verdaccio/apache_proxy below this example? I think will help a lot of people. We might do the same for nginx

Reply:
It looks exactly the same. It'd be less confusing to have just one canonical example, otherwise people are wasting their time looking for "but what's the difference?".

Maybe we just display a link to docker-examples conf here?

Reply:
Yes, exactly, that what I meant. 🙃